src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -77,14 +78,18 @@ export function LoadingTokenTable({ rowCount = PAGE_SIZE }: { rowCount?: number
 
 export default function TokenTable({ setRowCount }: { setRowCount: (c: number) => void }) {
   // TODO: consider moving prefetched call into app.tsx and passing it here, use a preloaded call & updated on interval every 60s
+  // TODO: consider passing `chainId` and `chainName` from parent, where we already have it resolved
   const chainName = validateUrlChainParam(useParams<{ chainName?: string }>().chainName)
   const { tokens, sparklines } = useTopTokens(chainName)
   setRowCount(tokens?.length ?? PAGE_SIZE)
 
+  const chainId = chainBackendNameToId(chainName)
+
   /* loading and error state */
   if (!tokens) {
     return (
       <NoTokensState
+        chainId={chainId}
```

Comment:
we may want the wallet's chain id not this one

Reply:
setting this to wallet's chain id would give a wrong glow effect when you change chain from the filters above the table. This corresponds to whatever is currently being displayed by the table, making sure we don't show e.g. Polygons glow for Celo etc.